<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Configuration</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
       <item row="0" column="0">
        <widget class="QLabel" name="serialLabel">
         <property name="text">
          <string>Serial port</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="serialCombo">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string notr="true">/dev/ttyACM0</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="dataGroup">
      <property name="title">
       <string>Data</string>
      </property>
      <layout class="QGridLayout" name="dataLayout" rowstretch="0,0,0">
       <item row="0" column="1">
        <widget class="QLabel" name="dotLamp">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="text">
          <string>##</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="dashLamp">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="text">
          <string>##</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="keyLabel">
         <property name="text">
          <string>Key</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Chart</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionFileExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Config">
    <property name="title">
     <string>&amp;Run</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRunOpen"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Config"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRunOpen"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHelpAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionFileExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionRunOpen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open serial port</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
